rj_gameplay/rj_gameplay/tactic/runner_tactic.py

```diff
@@ -0,0 +1,67 @@
+from typing import List, Tuple
+
+import stp
+from rj_msgs.msg import RobotIntent
+
+from rj_gameplay.role import runner_role
+
+
+class RunnerTactic(stp.tactic.Tactic):
+    """Wrapper for the Runner Role that handles assigning said role to whichever Robot is our goalie."""
+
+    def __init__(self, world_state: stp.rc.WorldState, runner_id: int):
+        """Special case where we want only robot 1 to be runner, from init."""
+        super().__init__(world_state)
+
+        # TODO: rather than passing in hardcoded goalie id on init, gameplay node should
+        #       have a set robot and pass that (in case robot 0 is ejected or broken)
+        self._role_requests.append(
+            (stp.role.cost.PickRobotById(runner_id), runner_role.RunnerRole)
+        )
+
+    def init_roles(self, world_state: stp.rc.WorldState) -> None:
+        # only has one role, but it's easier to copy-paste the structure
+        for i, robot in enumerate(self.assigned_robots):
+            role = self._role_requests[i][1]
+            if role is runner_role.RunnerRole:
+                self.assigned_roles.append(role(robot))
+
+    def tick(
+        self, world_state: stp.rc.WorldState
+    ) -> List[Tuple[int, RobotIntent]]:  # (id, intent)
+    
+#        corners = []
```

Comment:
Same comment as above. According to the Contributions Page, I would include comments on what you are making.